src/float.rs

```diff
@@ -5,15 +5,15 @@ use core::ops::{Add, Div, Neg};
 use core::f32;
 use core::f64;
 
-use {Num, NumCast, ToPrimitive};
+use {Num, NumCast, ToFromBytes, ToPrimitive};
 
 #[cfg(all(not(feature = "std"), feature = "libm"))]
 use libm;
 
 /// Generic trait for floating point numbers that works with `no_std`.
 ///
 /// This trait implements a subset of the `Float` trait.
-pub trait FloatCore: Num + NumCast + Neg<Output = Self> + PartialOrd + Copy {
+pub trait FloatCore: Num + NumCast + Neg<Output = Self> + PartialOrd + Copy + ToFromBytes {
```

Comment:
It's a breaking change to add requirements to the existing traits. `ToFromBytes` will have to stand alone.